readthedocs/settings/docker_compose.py

```diff
@@ -57,6 +57,15 @@ def LOGGING(self):
                 'handlers': ['null'],
                 'propagate': False,
             },
+            'azure.storage.common._auth': {
```

Comment:
I have this patched locally, and just did 
```suggestion
            'azure.storage.common': {
```

I think that will solve this in the future.